editor/src/components/canvas/canvas-component-entry.tsx

```diff
@@ -75,18 +76,18 @@ export const CanvasComponentEntry = React.memo((props: CanvasComponentEntryProps
     clearRuntimeErrors()
   }, [clearRuntimeErrors])
 
+  const containerRef = useApplyCanvasOffset(canvasProps?.scale ?? null)
+
   if (canvasProps == null) {
     return <CanvasLoadingScreen />
   } else {
     return (
       <div
         id='canvas-container-outer'
+        ref={containerRef}
         style={{
           position: 'absolute',
           zoom: canvasProps.scale >= 1 ? `${canvasProps.scale * 100}%` : 1,
```

Comment:
I think zoom and scale should be set at the same place, now one file is setting scale if scale < 1, and another file is setting zoom if scale > 1

Reply:
fixed